lib/charms/loki_k8s/v0/loki_push_api.py

```diff
@@ -166,6 +166,31 @@ def __init__(self, *args):
   lib will automatically resolve the relation name inspecting the using the
   meta information of the charm
 
+Anytime the relation between a Loki provider charm and a Loki consumer charm is
+established a `LokiPushApiEndpointJoined` event is fired. In the consumer side
+is it possible to observe this event with:
+
```

Comment:
I'm curious why should charm code care about these events. Maybe worth adding to docstring?

Reply:
We need to add or remove configuration in the consumer side (grafana-agent-operator).
Obviously docstrings can be improved!
Take a look at OP-369